elao.app.docker/Makefile.dist

```diff
@@ -35,16 +36,67 @@ install@staging:
 	# # Composer
 	# composer install --ansi --verbose --no-interaction --no-progress --prefer-dist --optimize-autoloader --no-scripts
 	# # Npm
-	# npm install --color=always --no-progress --no-audit
+	# npm install --color=always --no-progress --no-audit --no-fund
 	# # Yarn
 	# yarn install --color=always --no-progress
 
+############
+# Database #
+############
+
+## Database - Init (create+schema)
+db.install: db.create db.update-force
+
+## Database - Create (db only)
+db.create:
+	symfony console doctrine:database:create --if-not-exists
+
+## Database - Migrate (run migrations)
+db.migrate:
+	# If you use Messenger, without auto-setup nor a migration:
+	# Messenger:
+	# symfony console messenger:setup-transports
+	# Doctrine:
+	symfony console doctrine:migrations:sync-metadata-storage
+	symfony console doctrine:migration:migrate --no-interaction
+
+## Database - Generate a migration from current db schema diff
+db.diff:
+	symfony console doctrine:migrations:diff
+
+## Database - Update (alias for migrate)
+db.update: db.migrate
```

Comment:
If the migrations are not already setup at this point for the project, this alias can be replaced with `db.update: db.update-force`.